docs/src/examples/components/Icon/Variations/IconAnimated.shorthand.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Icon } from '@stardust-ui/react'
+
+class IconExample extends React.Component {
+  state = {
+    playState: 'running',
+  }
+  pauseTimeout
+  runningTimeout
+  componentDidMount() {
+    // after 10s pause the animation
+    this.pauseTimeout = setTimeout(() => {
+      this.setState({ playState: 'paused' })
+    }, 10000)
+    // after 20s run the animation again
+    this.runningTimeout = setTimeout(() => {
+      this.setState({ playState: 'running' })
+    }, 20000)
+  }
```

Comment:
Let's simplify this example to not use timeouts.  Also, move it to the Transition examples for now.  We won't show individual component transitions just yet.